Sudoku/Controller/GameController.cs

```diff
@@ -12,25 +13,27 @@ public class GameController
     private readonly Game _game;
     private readonly IBoardView _boardView;
     private readonly IVisitorFactory _visitorFactory;
+    public MainController Controller { get; set; }
 
     public GameController(Game game, IBoardView view, IVisitorFactory visitorFactory)
     {
         _game = game;
         _game.Controller = this;
         _boardView = view;
         _visitorFactory = visitorFactory;
-        _boardView.Controller(this);
     }
 
-    public void RunGame(AbstractBoard board)
+
+    public bool RunGame(AbstractBoard board)
     {
+        var gameOver = false;
         _game.Board = board;
 
         _boardView.Accept(_visitorFactory.Create(DotNetEnv.Env.GetString("UI")));
 
         Console.WriteLine("Starting your Sudoku game. Press ESC to quit the game.");
```

Comment:
ik hoop niet dat ze moeilijk gaan doen over view dingetjes in een controller

Reply:
goed punt. Ik fix het